src/components/challenges/active.tsx

```diff
@@ -12,70 +11,64 @@ type ActiveProps = {
     start_date: string;
     end_date: string;
     participants: {
-      user_id: string,
-      first_name: string,
-      last_name: string,
-      yoe: number,
-      company: string,
-      designation: string,
-      img: string,
-      github_id: string,
-      linkedin_id: string,
-      twitter_id: string,
-      instagram_id: string,
-      is_member: number,
-      rds_member_id: string
+      user_id: string;
+      first_name: string;
+      last_name: string;
+      yoe: number;
+      company: string;
+      designation: string;
+      img: string;
+      github_id: string;
+      linkedin_id: string;
+      twitter_id: string;
+      instagram_id: string;
+      is_member: number;
+      rds_member_id: string;
     }[];
     is_active: number;
     is_user_subscribed: number;
   };
 };
 
-const Active: FunctionComponent<ActiveProps> = ({ content }) => {
+const Active: FC<ActiveProps> = ({ content }) => {
   const [state, setState] = useState({
     is_user_subscribed: content.is_user_subscribed,
   });
 
```

Comment:
Since the state object here has only property, we can define the state like this
`const [is_user_subscribed, setUserSubscribed] = useState(content.is_user_subscribed)`